core/controllers/profile_test.py

```diff
@@ -200,7 +196,7 @@ def test_user_settings_of_non_existing_user(self):
             u'username': None,
         }
         response = self.get_json(feconf.SIGNUP_DATA_URL)
-        self.assertEqual(response, values_dict)
+        self.assertDictContainsSubset(response, values_dict)
```

Comment:
We can't equal the dictionaries because CSFR token is different every time?

Reply:
Yes